src/orion/core/worker/trial.py

```diff
@@ -141,7 +141,7 @@ class Param(Value):
 
         allowed_types = ('integer', 'real', 'categorical')
 
-    __slots__ = ('experiment', '_id', '_status', 'worker',
+    __slots__ = ('experiment', '_status', 'worker', '_id',
```

Comment:
?

Reply:
Also, it seems that worker is not used at all. But that's a general comment about trials, independent of this PR's intention. Should it be removed, to be re-introduced whenif needed?

Reply:
I would leave it there since it's not related to this PR.

Reply:
Probably, is it even set somewhere beside the test trials?